src/main.css

```diff
@@ -252,6 +252,7 @@ a {
     top: 0;
     left: 0;
     pointer-events: none;
+    z-index: 100;
```

Comment:
This layers the popovers on top of the header, so for trains at the top of the line, you can still see the details correctly.